examples/with-tailwind/packages/ui/tsup.config.ts

```diff
@@ -1,13 +1,46 @@
 import { defineConfig, Options } from "tsup";
+import path from "path";
+import fs from "fs-extra";
+
+const jsFileScan = (dir:string):string[] => {
+  
+  const paths = []
+  const nodes = fs.readdirSync(dir);
+  for (const node of nodes) {
+    const nodePath = path.join(dir, node);
+    const stat = fs.statSync(nodePath);
+    if (stat.isDirectory()) {
+      paths.push(...jsFileScan(nodePath));
+    } else if (stat.isFile() && (node.endsWith(".js") || node.endsWith(".mjs"))) {
+      paths.push(nodePath);
+    }
+  }
+
+  return paths;
+}
 
 export default defineConfig((options: Options) => ({
   treeshake: true,
   splitting: true,
   entry: ["src/**/*.tsx"],
-  format: ["esm"],
+  outDir: 'dist',
+  format: ['cjs'],
```

Comment:
I'd love to be recommending the most modern practices we can. Any way we can get this back to ESM?

Reply:
resolved in https://github.com/vercel/turbo/pull/6307/commits/c79b74eb86168c002b08bb0860304a572dd33fc1